### PR TITLE
Update guides/source/plugins.md [ci skip]

### PR DESCRIPTION
**Pull request by bogdanvlviv**

### CODE REVIEW

File: guides/source/plugins.md [hunk 1]
```diff
@@ -458,13 +449,13 @@ Gem plugins currently in development can easily be shared from any Git repositor
 commit the code to a Git repository (like GitHub) and add a line to the Gemfile of the application in question:
 
 ```ruby
-gem 'yaffle', git: 'git://github.com/yaffle_watcher/yaffle.git'
+gem "yaffle", git: "git://github.com/rails/yaffle.git"
```
Comment (yhirano55) [inline guides/source/plugins.md]: @bogdanvlviv Could you change `https://`? And I'll close a request #30262.

Comment (bogdanvlviv) [inline guides/source/plugins.md]: @yhirano55 Thanks

File: guides/source/plugins.md [hunk 2]
```diff
@@ -458,13 +449,13 @@ Gem plugins currently in development can easily be shared from any Git repositor
 commit the code to a Git repository (like GitHub) and add a line to the Gemfile of the application in question:
 
 ```ruby
-gem 'yaffle', git: 'git://github.com/yaffle_watcher/yaffle.git'
+gem "yaffle", git: "https://github.com/rails/yaffle.git"
```
Comment (rafaelfranca) [inline guides/source/plugins.md]: can we keep the link outside the `rails` org?

Comment (bogdanvlviv) [inline guides/source/plugins.md]: I've changed it guided by that one day there might be a user by https://github.com/yaffle_watcher

**Pull request merged**